### ISSUE TITLE
Netural status is no longer supported by Github

### ISSUE THREAD

**Issue by tannerbaum** [mheap/github-action-required-labels#31]
Unfortunately it seems like Github dropped the neutral exit code a year or two back and have yet to replace it with anything, which is quite annoying. 

https://github.com/actions/runner/issues/662
https://github.community/t/github-actions-neutral-exit-code-is-incorrectly-interpreted-as-failure/16088

It is still in the actions-toolkit because that package hasn't been updated in a couple of years either. 

This was a bummer because I was really looking forward to the possibility of having a neutral exit like @moritzjacobs originally suggested. I think I will be able to manage without it though.

**Comment by mheap** [comment]
This is resolved in #32, which has just been released and is available as `@v2`.

There is now an output that you can use to simulate the neutral exit code:

```yaml
name: Pull Request Labels
on:
  pull_request:
    types: [opened, labeled, unlabeled, synchronize]
jobs:
  label:
    runs-on: ubuntu-latest
    outputs:
      status: ${{ steps.check-labels.outputs.status }}
    steps:
      - id: check-labels
        uses: mheap/github-action-required-labels@v2
        with:
          mode: exactly
          count: 1
          labels: "semver:patch, semver:minor, semver:major"
          exit_type: success
  do-other:
    runs-on: ubuntu-latest
    needs: label
    steps:
      - run: echo SUCCESS
        if: needs.label.outputs.status == 'success'
      - run: echo FAILURE && exit 1
        if: needs.label.outputs.status == 'failure'
```

**Comment by tannerbaum** [comment]
@mheap this is beautiful, thanks!
